src/ctim/schemas/vocabularies.cljc

```diff
@@ -335,7 +335,12 @@
     "email"
     "imei"
     "imsi"
-    "amp-device"})
+    "amp-device"
```

Comment:
using a dash is not coherent with the new observable types that use an underline char.